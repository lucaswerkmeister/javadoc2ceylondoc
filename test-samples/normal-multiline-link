/**
 * line 1
 *
 * {@link java.lang.String}
 * {@link File.Appender#write() write method}
 * {@link java.lang.Integer#parseInt(String s) text to int}
 * {@link MySingleton.instance}
 */
