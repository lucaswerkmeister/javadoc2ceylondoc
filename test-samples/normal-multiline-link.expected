"line 1
 
 [[java.lang::String]]
 [[write method|File.Appender.write]]
 [[text to int|java.lang::Integer.parseInt]]
 [[MySingleton.instance]]"
